### ISSUE TITLE
Program exit with code -1073740940 (0xC0000374)

### ISSUE THREAD

**Issue by lsylusiyao** [SciSharp/Keras.NET#126]
Dear Keras.NET:
I've using the keras.NET for days writing DQN, which needs to run `Fit()` and `Predict()` in loops for tens of thousand times. It can run normally when it starts. 
However, after 6 or 7 mins ( guess functions have be called hundreds of thousand times ), the program crashes with NO EXCEPTION and code `-1073740940` ( equal to `0xC0000374` in hex, meaning sth wrong inside the heap ). I tried using source codes of `Keras.NET` and `Numpy.NET` but showed nothing. 
Even using source code of python_net standard and clicked every exceptions I could find, VS gave me exceptions of `System.FileNotFoundException` when program started, where models like `org.python`, `pandas`(I don't need it actually) were not found. When entering my main loops, everything worked well. 

![image](https://user-images.githubusercontent.com/26361428/83320143-268ffd00-a277-11ea-887d-b316cc50e620.png)

After several minutes, program crashed. VS exited to the normal code mode with no message but below. 

![image](https://user-images.githubusercontent.com/26361428/83320156-53dcab00-a277-11ea-91a6-16468ba4fbd8.png)

The program printed like below.

![image](https://user-images.githubusercontent.com/26361428/83320172-8090c280-a277-11ea-956e-daa9a27d7063.png)

I'm really confused because till now I don't know where to get start. Could you please help me? 
Thanks a lot.

**Comment by deepakkumar1984** [comment]
This is wierd, seesm like its unable to load other dlls other than keras. Keras is not dependend on any on the dll you show in the error. Could be this other issue with your VS installation

**Comment by lsylusiyao** [comment]
However, that's not the only one computer with VS2019 that did like this. 😭

**Comment by eduardobellotto** [comment]
This is the same issue I've found in this [issue](https://github.com/SciSharp/Keras.NET/issues/110).
Unfortunately Keras.NET just works for a single fit method use.

**Comment by lsylusiyao** [comment]
Wow, that's interesting. During the last week I've found out a way to solve the problem that I wrote the DQN by TF2 with Keras (actually just the Keras inside the TF2) on Python, and the environment (which is a little complicated so I don't want to rewrite it by Python) still by C#, and connected them through gRPC.😀
For now it works well. Maybe for a period of time I will use this since the Python version is more stable. Thank you for helping me.😀
